packages/cli/src/lib/bundler/config.ts

```diff
@@ -232,12 +233,50 @@ export async function createConfig(
     require.resolve('react-refresh'),
   ];
 
+  const mode = isDev ? 'development' : 'production';
+  const optimization = optimizationConfig(options);
+
+  if (
```

Comment:
Should this path be skipped if more is already `development`?

Reply:
Yep, probably makes sense! Our code should have exactly the same result as the default behavior when mode is 'development', but of course "should" is the operative word 😁.

Reply:
Done in 232b134606d1.

Reply:
This is actually a nice improvement, because previously the warning would be printed even when it wasn't relevant.